### PR TITLE
[IMP] website: introduce `s_images_mosaic` snippet

### PR DESCRIPTION
**Pull request by Brieuc-brd**
This commit adds the new s_images_mosaic snippet.

task-4105446
Part of task-4077427

Requires:
- https://github.com/odoo/design-themes/pull/865

Capture d’écran 2024-08-13 à 11 41 50

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Brieuc-brd** [comment]
@stefanorigano 👋 
Here's a new snippet ready for review, could you have a look at it ?
Thanks in advance !

**Comment by Brieuc-brd** [comment]
⚠️ **Note for the tech. review** ⚠️ :
This snippet doesn't provide fallback (yet) images for industries because it needs square images, otherwise the rendering won't be good, for example :
| Current | Expected |
|--------|--------|
| Capture d’écran 2024-08-13 à 16 29 40 | Capture d’écran 2024-08-13 à 16 30 38 |

The web builder provides the "Stretch" feature, which may be the solution for forcing the images to always be square.
Capture d’écran 2024-08-13 à 16 32 12

But, as with the "Flip" feature, I don't think it can be set by default. What do you think?

**Comment by bso-odoo** [comment]
> But, as with the "Flip" feature, I don't think it can be set by default. What do you think?

Unfortunately this PR has not been merged yet. <https://github.com/odoo/odoo/pull/94187> It might have made it possible to use any image option.

File: addons/website/views/snippets/s_images_mosaic.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_images_mosaic" name="Images Mosaic">
+    <section class="s_images_mosaic pt72 pb72 o_cc o_cc5">
+        <div class="container">
+            <div class="row px-3">
+                <div class="col-lg-3">
```
Comment (stefanorigano) [inline addons/website/views/snippets/s_images_mosaic.xml:8]: I find the text column a bit too crowded at the initial LG steps ( `> LG` it's ok tho 👍 ).
I know that you had to find "a compromise", but I'd trade a bit of visual result on very large screen for more clarity on regular/common devices. Wdyt? Could you test?  
 
| This PR: `col-lg-3` | Proposal: `col-lg-4` (+ remove sibling's `offset-lg-1`) |
|--------|--------|
| image | image |

**Review by stefanorigano — APPROVED** [review]
Hello @Brieuc-brd , design LGTM 👍 

I added a note about a small improvement for the layout: not blocking but worth testing IMO 😉 Good luck for the  tech. review.💪

**Comment by Brieuc-brd** [comment]
- Reviewed default text
- Added custom texts for themes
- Rebased

**Comment by Brieuc-brd** [comment]
Fixed heading display class : https://github.com/odoo/odoo/compare/1341fcb28c544d8a819aab7fd9328acfe1526bcc..26468564ed5f1f4d62545a49aa13054a4000edad

**Review by SergeBayet — APPROVED** [review]
LGTM! Thanks
PR queued and will be r+ after https://github.com/odoo/odoo/pull/177620

**Comment by SergeBayet** [comment]
Commit(s) moved into
- https://github.com/odoo/odoo/pull/180736

**Pull request closed without merging**